src/driver-hidpp20.c

```diff
@@ -719,6 +829,14 @@ hidpp20drv_init_feature(struct ratbag_device *device, uint16_t feature)
 	}
 	case HIDPP_PAGE_COLOR_LED_EFFECTS: {
 		log_debug(ratbag, "device has color effects\n");
+		drv_data->capabilities |= HIDPP_CAP_COLOR_LED_EFFECTS_8070;
+
+		/* we read the profile once to get the correct number of
+		 * supported leds. */
+		if (hidpp20drv_read_color_leds(device))
+      return 0;
+
```

Comment:
fix the indentation here please. Wish there was a way to just do that online and merge it, but if there is I don't know how :)